plugins/xss.js

```diff
@@ -17,6 +17,7 @@ const options = {
     input: ['checked', 'disabled', 'type'],
     iframe: ['width', 'height', 'allowfullscreen', 'frameborder'],
     img: [...xss.whiteList.img, 'style'],
+    a: ['href', 'rel'],
```

Comment:
Is this intended to overwrite [the default for the a tag](https://github.com/leizongmin/js-xss/blob/master/lib/default.js#L13)?
```suggestion
    a: [...xss.whiteList.a, 'rel'],
```